tests/test_views.py

```diff
@@ -101,6 +102,15 @@ def test_get_succeeds(self):
         assert response.status_code == status.HTTP_200_OK
         assert response.data == {'method': 'GET'}
 
+    def test_logged_in_get_succeeds(self):
+        user = User.objects.create_user('user', 'user@example.com', 'password')
+        request = factory.get('/')
+        del user.is_active
```

Comment:
why do you delete is_active?

Reply:
This is to force the ORM to make a query and uncover the issue. I could not reproduce the issue some other way.

Reply:
Since this seems to be a hack to cause an exception to occur, I recommend adding a comment to let future on-lookers know what's going on here.

Reply:
I added comment on [adrf](https://github.com/em1208/adrf) code.